src/TopBar/TopBar.js

```diff
@@ -24,11 +24,12 @@ const TopBar = ({ history }) => {
     <div className="top_expander">
       <div className="top_menu">
         <LookupControl onQueryChange={e => setQuery(e.target.value)} />
-        {/*}
-        <div className="top_menu_item">Naturtyper</div>
-        <div className="top_menu_item">Landskap</div>
-        <div className="top_menu_item">Statistikk for Art</div>
-  */}
+        <img
+          src="http://localhost:3000/logoer/adb32.png"
```

Comment:
Hvis du utelater protokoll og hostnavn og bare bruker relativ path til bildet så fungerer det også på andre maskiner
```suggestion
          src="/adb32.png"
```